### PR TITLE
[MM-27214] Cypress Automation Auth Numero Dos 2

### PR DESCRIPTION
**Pull request by hahmadia**
#### Summary

Automate Cypress Test

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-27214

image

image

### CODE REVIEW

**Review by anurag6713 — APPROVED** [review]

File: e2e/cypress/integration/enterprise/system_console/authentication/authentication2_spec.js [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+import * as TIMEOUTS from '../../../../fixtures/timeouts';
+
+import {getEmailUrl, getRandomId} from '../../../../utils';
+
+describe('Authentication Part 2', () => {
+    let testUser;
+    let testUserAlreadyInTeam;
+    let testTeam;
+
+    before(() => {
```
Comment (josephbaylon) [inline e2e/cypress/integration/enterprise/system_console/authentication/authentication2_spec.js:21]:
```suggestion
    before(() => {
        cy.apiRequireLicense();
        
```

**Review by josephbaylon — APPROVED** [review]
LGTM. Just minor suggestions below. Thanks @hahmadia !!

File: e2e/cypress/integration/enterprise/system_console/authentication/authentication2_spec.js [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+        cy.apiRequireLicense();
+
```
Comment (josephbaylon) [inline e2e/cypress/integration/enterprise/system_console/authentication/authentication2_spec.js:23]: Fix lint issue
```suggestion

```

**Review by furqanmlk — APPROVED** [review]
All test cases passed and look good to me

**Pull request merged**